package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "xliff-simple-merge",
-  "version": "1.1.2",
+  "version": "1.2.0",
```

Comment:
this could/should be reverted, as versions are updated on release using `np`

Reply:
ok